scripts/datastore.js

```diff
@@ -0,0 +1,36 @@
+// Imports the Google Cloud client library
+const Datastore = require('@google-cloud/datastore');
```

Comment:
This code seems like it belongs on the server side rather than the client side.
`require('@google-cloud/datastore');` for example, is a NodeJS syntax instead of vanilla JavaScript ran on the browser.

I think it would be better to put this in `/modules` directory so it's clear that this is server-side code.

Reply:
I see. So I can put this file under the @google-cloud/datastore folder. Sounds good